commons/deployment/src/main/java/com/github/mcollovati/quarkus/hilla/deployment/QuarkusHillaExtensionProcessor.java

```diff
@@ -284,12 +284,26 @@ void registerEndpoints(
     }
 
     @BuildStep
-    void registerHillaPushServlet(BuildProducer<ServletBuildItem> servletProducer, NativeConfig nativeConfig) {
+    void registerHillaPushServlet(
+            BuildProducer<ServletBuildItem> servletProducer,
+            QuarkusEndpointConfiguration endpointConfiguration,
+            NativeConfig nativeConfig) {
         ServletBuildItem.Builder builder = ServletBuildItem.builder(
                 QuarkusAtmosphereServlet.class.getName(), QuarkusAtmosphereServlet.class.getName());
```

Comment:
Same name twice? Don't know the signature yet.

Reply:
Can you please elaborate a bit? I'm not sure I understood the comment

Reply:
Now I see. It is correct. The first is the name of the servlet, the second is the servlet class.